### DISCUSSION TITLE
LokiStream v1.1.0

### DISCUSSION THREAD

**Discussion by Lavish883** [Lavish883/LokiStream-Lite#5] [Announcements]
Latest release of the LokiStream App, fixed a lot of minor bugs. No major changes. Got any Ideas to make it better?? Let me know 

<hr /><em>This discussion was created from the release <a href='https://github.com/Lavish883/LokiStream-Lite/releases/tag/v1.1.0'>LokiStream v1.1.0 </a>.</em>

**Comment by iCeY3** [comment]
Thanks for the cool app bro
As for me id like to have multiple sources like 9anime, allanime and zoro
Have MAL library sync

These are my top priorities

Thanks again

**Comment by Lavish883** [comment]
@iCeY3 Thank you for the feedback, I will look into the MAL library sync and try to implement it asap. The multiple sources might take a little more time.

P.S Share the app with your freinds and help it grow

**Comment by Akibsaifullah** [comment]
Can it run on android TV
